src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-size: 10px;
+  line-height: 14px;
+  --color1: #616070;
+  --color2: #00acdc;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color1);
+  margin: 0 0 16px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  margin-right: 4px;
+  display: block;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
+
```

Comment:
put here all styles from `stars` task. so we would can to change the number of active stars, by just changing the class name